third_party/valijson/valijson/compat/optional.hpp

```diff
@@ -0,0 +1,1042 @@
+// Copyright (C) 2011 - 2012 Andrzej Krzemienski.
```

Comment:
Not sure why this file was non-existing in the current E+ develop and whether it should have been comitted here.